packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js

```diff
@@ -16,20 +16,24 @@ const StyledAnchor = styled(Anchor)`
     text-decoration: none;
     border-color: white;
   }
-  ${props => props.isActive && `
+  ${props =>
+    props.isActive &&
+    `
     border-color: white;
   `}
 `
 
-function NavLink (props) {
+function NavLink(props) {
   const { href, router, text } = props
   const isActive = router.asPath.includes(href)
   return (
-    <StyledAnchor href={href} isActive={isActive}>
-      <StyledSpacedText color='white' weight='bold'>
-        {text}
-      </StyledSpacedText>
-    </StyledAnchor>
+    <Box as="li">
```

Comment:
I'd rather this was moved to the consuming component, as I don't want `NavLink` to make assumptions on how it's rendered

Reply:
This will also fix the broken test